content/versions/v4.0.0/deploy/cluster-sizing.md

```diff
@@ -0,0 +1,98 @@
+| `libre-core`           | 3                           | 1            | 2              | No          | N/A            | HA requires 2 pods, but 3 is to avoid hotkey issues and balance load |
+| `bpmn-engine`          | 3                           | 1            | 2              | No          | N/A            | HA requires 2 pods, but 3 is to avoid hotkey issues and balance load |
+| `redpanda`             | 3                           |              |                | Yes         | 100            | High IOPS                                                            |
+| `libre-audit`          | 2                           | 1            | 1              | No          | N/A            |                                                                      |
+| `libre-audit-postgres` | 2                           | 1            | 2              | Yes         | 250            | Runs in pod with `libre-audit`                                       |
+| `keycloak`             | 2                           | 1            | 2              | No          | N/A            |                                                                      |
+| `keycloak-postgres`    | 2                           | 1            | 2              | No          | 200            | Runs in pod with `keycloak`                                          |
+| `router`               | 2                           | 1            | 2              | Yes         | <1             | Requires volume to compose supergraph                                |
+| `quest-db`             | 1                           | 4            | 8              | Yes         | 250GB          | High Throughput and IPOS                                             |
+| `restate`              | 3                           |              |                | Yes         | 50             | High Throughput and IPOS                                             |
+| `appsmith`             | 3                           | 4            |                | Yes         | 50             | High Throughput and IPOS                                             |
+| `grafana`*             | 3                           | 0.5          | 2              | No          | 20-50          | Storage can be in host or in object bucket.                          |
+
+ * May run [in separate cluster](#monitoring-stack)
+
+### Monitoring stack
+
+The following table provides minimal compute recommendations for the monitoring stack.
+
+The default recommendation is to run your Rhize observability stack in the nodes that also run the Rhize application.
+However, some deployments prefer to separate monitoring to its own cluster.
+
+| Service                 | Pods for HA (replica count) | vCPU cores per pod | Memory per pod | DiskSize (GiB) |
+|-------------------------|-----------------------------|--------------------|----------------|----------------|
```

Comment:
Might need to checking with the devops guys if this is still applicable in the latest tempo/loki/prom distributed helm-charts.